satellite/console/consoleweb/consoleql/query.go

```diff
@@ -4,8 +4,13 @@
 package consoleql
 
 import (
+	"errors"
+	"fmt"
+
 	"github.com/graphql-go/graphql"
 	"github.com/skyrings/skyring-common/tools/uuid"
+	"storj.io/storj/internal/post"
```

Comment:
goimport this or just add blank line between github and storj imports

Reply:
you also have unneeded blank line before `"storj.io/storj/satellite/console"`

Reply:
Got it, thanks